resources/resources.py

```diff
@@ -3,6 +3,7 @@
 from data.filter_options import FilterOptions
```

Comment:
I think you can safely remove this file from your PR

Reply:
Right, just saw that Dom has the same code. I'll wait for his PR to be merged and then I can deal with the merge conflicts.